### PR TITLE
fix service responses for behaviors

### PR DESCRIPTION
**Pull request by floweisshardt**
solves #366

### CODE REVIEW

**Comment by floweisshardt** [comment]
Needs to be done for cob4-1 too

File: cob_default_robot_behavior/scripts/trigger_srvs_cob4-2.py [hunk 1]
```diff
@@ -133,22 +137,22 @@ def pick_cb(req):
...
     sss.set_light("light_base","red")
     sss.set_light("light_torso","red")
-    return
+    return TriggerResponse(True, "")
     
 def setLightGreen_cb(req):
     sss.set_light("light_base","green")
     sss.set_light("light_torso","green")
-    return
+    return TriggerResponse(True, "")()
```
Comment (fmessmer) [inline cob_default_robot_behavior/scripts/trigger_srvs_cob4-2.py:155]: remove `()`

File: cob_default_robot_behavior/scripts/trigger_srvs_cob4-2.py [hunk 2]
```diff
@@ -74,8 +74,10 @@ def torso_front_cb(req):
     if handle_arm_left.get_error_code() == 0 and handle_arm_right.get_error_code() == 0:
         sss.move_base_rel("base",[0,0,1.57],False)
         sss.move("torso","front",True)
+    else:
+        TriggerResponse(False, "Could not move arms.")
```
Comment (fmessmer) [inline cob_default_robot_behavior/scripts/trigger_srvs_cob4-2.py:78]: `return` missing

File: cob_default_robot_behavior/scripts/trigger_srvs_cob4-2.py [hunk 3]
```diff
@@ -85,7 +87,9 @@ def front_to_home_cb(req):
     if handle_arm_left.get_error_code() == 0 and handle_arm_right.get_error_code() == 0:
         sss.move_base_rel("base",[0,0,-1.57],False)
         sss.move("torso","home",False)
-    return
+    else:
+        TriggerResponse(False, "Could not move arms.")
```
Comment (fmessmer) [inline cob_default_robot_behavior/scripts/trigger_srvs_cob4-2.py:91]: `return` missing

**Comment by fmessmer** [comment]
all remarks are addressed in #406

**Pull request merged**